TaskModel.json

```diff
@@ -3021,6 +3021,87 @@
 			"neededCapacity": [
 				{ "name": "oil", "value": 720 }
 			]
+		},
+		{
+			"group": 1,
+			"name": "Containerzug von Krefeld nach Helsingborg ",
+			"service": 11,
+			"descriptions": [
+				"Verbinde das Rheinland mit dem Norden."
+			],
+			"stations": [ "KKR", "XVH" ],
+			"neededCapacity": [
+				{ "name": "containers", "value": 72 }
+			]
+		},
+		{
+			"group": 1,
+			"name": "Güterzug von Hamburg nach Malmö ",
+			"service": 11,
+			"descriptions": [
+				"Bringe einen Kombinierten Güterzug nach Malmö."
+			],
+			"stations": [ "AHAR", "XVM" ],
+			"neededCapacity": [
+				{ "name": "containers", "value": 20 },
+				{ "name": "oil", "value": 660 },
+				{ "name": "wood", "value": 600 },
+				{ "name": "cars", "value": 60 }
+			]
+		},
+		{
+			"group": 1,
+			"name": "Schwerer Intermodaltransport von von %s nach %s ",
+			"service": 11,
+			"descriptions": [
+				"Verbinde %s mit %s mit diesem Intermodalen Güterzug."
+			],
+			"stations": [ ],
+			"neededCapacity": [
+				{ "name": "containers", "value": 72 }
+			]
+		},
+		{
+			"group": 1,
+			"name": "Güterzug von von %s nach %s ",
+			"service": 11,
+			"descriptions": [
+				"Verbinde %s mit %s mit diesem Güterzug."
+			],
+			"stations": [ ],
+			"neededCapacity": [
+				{ "name": "containers", "value": 20 },
+				{ "name": "oil", "value": 660 },
+				{ "name": "wood", "value": 600 },
+				{ "name": "cars", "value": 60 }
+			]
+		},
+		{
+			"group": 1,
+			"name": "Öresundståg von XDKH nach XVH",
+			"service": 2,
+			"descriptions": [
+				"Verbinde Kopenhagen mit Helsingborg.",
+				"Fahre den Zug zwischen Dänemark und Schweden."
+			],
+			"stations": [ "XDKH", "XDKHL", "XVM", "🇸🇪KL", "🇸🇪LK", "XVH" ],
+			"neededCapacity": [
+				{ "name": "passengers", "value": 0 }
+			]
+		},
+		{
+			"group": 1,
+			"name": "Öresundståg von XVH nach XDKH",
```

Comment:
Gleiches wie in den vorherigen Kommentaren

Reply:
Ich habe eigentlich bei denen, die Fix sind die Route mit vorgegeben, aber auch welche Hinzugefügt, die nicht Fix sind, und das dort offen gelassen. Wenn das aber besser ist mit nichts Fixen at all, dann ändere ich das nochmal

Reply:
Wenn du nur die Route mitgeben willst, musst du `"pathSuggestion"` statt `"stations"` verwenden 😅
Die `stations` werden alle als Halte, bzw. Start/Endpunkte genommen.

Reply:
Ja genau, das meine ich (Habe mich nur falsch ausgedrückt, entschuldige). Bei denen, denen ich einen festen Namen gegeben habe, haben doch auch einen festen Start und Endpunkt, also sollte das doch passen, oder? Ich habe auch nicht-feste Gz hinzugefügt, aber z.B: der Öresundstag, da habe ich ja als Anfangs und End-Punkt Kopenhagen und Helsingborg (oder andersrum, in die andere Richtung) ausgewählt, wieso sollte der dann wo anders fahren? Das verstehe ich nicht.
Wenn euch das aber lieber ist, dass da gar keine festen Aufträge mehr drin sind, kann ich das natürlich ändern